configurations/30-sl-nginx.entrypoint.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
```

Comment:
I wonder if we can use a web server that supports integration with LetsEncrypt so we can have SSL certificate? Some alternatives are on https://github.com/nginx-le/nginx-le

Reply:
Great find this nginx-le! Too bad I wasn't aware of this one a few months ago before a rewrite mine from scratch lol
Just one tip: it would be good security practive to avoid running front container as root. This involves using port >1024 but it's not a big deal as port remapping is easy from  docker-compose

Reply:
Hi @Metabaron1, thanks for the insight. I'm going to make some modifications according to your 2 points. Please stay tight.